x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/auto_follow_pattern_table/auto_follow_pattern_table.js

```diff
@@ -152,7 +152,7 @@ export const AutoFollowPatternTable = injectI18n(
             }),
             icon: 'pencil',
             onClick: ({ name }) => {
-              editAutoFollowPattern(name);
+              selectAutoFollowPattern(name);
```

Comment:
Why do we need to select the auto-follow pattern here? I just tried removing this line and the editing functionality seemed to work fine.

On a separate note, I think we may want to implement this the way we did in Remote Clusters, so that this is a link with an `href` instead of a button with an `onClick`. This way people could open this in a new tab or copy the URL if they wanted to for some reason: https://github.com/elastic/kibana/blob/master/x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/remote_cluster_table/remote_cluster_table.js#L183

Reply:
It works because, when there is no selection the edit component looks at the route param and fetches from the server (what happens on a page refresh). But this could be changed (that's how it was before you implemented the href on remote cluster). But I'll change it to an `href`, good point